extension/tree.html

```diff
@@ -0,0 +1,65 @@
+﻿<!DOCTYPE html>
+
+<html lang="en" xmlns="http://www.w3.org/1999/xhtml">
+<head>
+	<meta charset="utf-8" />
+	<title>DeepSite Source Tree</title>
+	<link href="css/bootstrap.min.css" rel="stylesheet">
+	<link href="css/tree.css" rel="stylesheet">
+
+	<script src="js/jquery-3.2.1.min.js"></script>
+</head>
+<body>
+	<!-- Canvas Element for Lines -->
+	<canvas class="canvas-element" id="canvas-element"></canvas>
+
+	<div class="nav-bar">
+		<div class="nav-bar-text-wrapper">
+			<img class="nav-bar-logo" src="images/icon128.png" />
+			<span class="nav-bar-deep">Deep</span><span class="nav-bar-cite">Cite</span>
+			<span>Source Tree</span>
+			<div class="nav-button-wrapper">
+				<div class="button-donate" onclick="donateButtonClicked()">
+					Donate
+				</div>
+			</div>
+		</div>
+	</div>
+	<div class="content">
+		<div class="tree-container" id="tree-container">
+			<!-- This information is dynamically populated -->
+		</div>
+	</div>
+
```

Comment:
The below components are hidden on-page, however they are utilized dynamically through deep cloning from the javascript.  Neato!